upf/upf_api.c

```diff
@@ -313,6 +313,67 @@ vl_api_upf_pfcp_format_t_handler (vl_api_upf_pfcp_reencode_t * mp)
     vec_free (s);
 }
 
+static void
+send_upf_nat_pool_details (vl_api_registration_t * reg,
+			   upf_nat_pool_t * np, u32 context)
+{
+  vl_api_upf_nat_pool_details_t *mp;
+  upf_main_t *sm = &upf_main;
+  upf_nat_addr_t *ap;
+  u8 *nwi_name = 0;
+  u32 name_len;
+  u32 max_users = 0;
+  u32 current_users = 0;
+
+  mp = vl_msg_api_alloc (sizeof (*mp));
+  clib_memset (mp, 0, sizeof (*mp));
+
+  mp->_vl_msg_id = htons (VL_API_UPF_NAT_POOL_DETAILS + sm->msg_id_base);
+  mp->context = context;
+
+  name_len = clib_min (vec_len (np->name) + 1, ARRAY_LEN (mp->name));
+  memcpy (mp->name, np->name, name_len - 1);
+  ASSERT (0 == mp->name[name_len - 1]);
```

Comment:
borrowing from dhcp_api.c , I find their version more readable:
```suggestion
  len = clib_min (sizeof (mp->name) - 1, vec_len (np->name));
  memcpy (mp->name, np->name, len);
  mp->name[len] = 0;
```

Reply:
Yep, this looks better. Changes applied for both pool name and nwi